components/DateInput/index.tsx

```diff
@@ -41,55 +40,57 @@ export default function DateInput({
     };
   }, []);
 
-  // Automatically show picker when dropdown icon is clicked
-  const handleDropdownClick = () => {
-    // Use requestAnimationFrame to ensure the input is rendered before showing picker
-    requestAnimationFrame(() => {
-      hiddenInputRef.current?.showPicker();
-    });
-  };
-
-  const formatDate = (dateString: string) => {
-    if (!dateString) return placeholder;
-
-    try {
-      return new Date(dateString).toLocaleDateString('en-US', {
-        year: 'numeric',
-        month: 'long',
-        day: 'numeric',
-      });
-    } catch {
-      return dateString;
-    }
-  };
-
   return (
     <Flex $direction="column" $gap="4px">
       {label && (
         <P2 as="label" htmlFor="date">
           {label}
         </P2>
       )}
-      <DateInputWrapper onClick={handleDropdownClick}>
-        <SelectContainer ref={containerRef}>
-          <P2 $color={COLORS.midgray}>{formatDate(value)}</P2>
-          <DropdownIcon onClick={handleDropdownClick}>
-            <Icon type="calendar" />
-          </DropdownIcon>
-
-          <HiddenDateInput
-            ref={hiddenInputRef}
-            type="date"
-            id="date"
-            value={value}
-            onChange={e => {
-              onChange(e.target.value);
-            }}
-            min={min}
-            max={max}
-          />
-        </SelectContainer>
-      </DateInputWrapper>
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        {/* config for Date Picker */}
+        <DatePicker
+          onChange={newValue => {
+            onChange(new Date(newValue?.toString() ?? '').toISOString());
```

Comment:
Maybe you don't want to run the onChange if newValue is null? Depends on your use case tho.